plugins/rollbar_dashboards/app/controllers/rollbar_dashboards/dashboards_controller.rb

```diff
@@ -2,54 +2,58 @@
 
 class RollbarDashboards::DashboardsController < ApplicationController
   NUM_ITEMS = 4
+  EXPIRES_IN = 1.minute
 
   def project_dashboard
     project = Project.find_by_permalink!(params.require(:project_id))
-
-    dashboard_html = Rails.cache.fetch(['project_dashboard', project], expires_in: 10.minutes) do
+    dashboard_html = Rails.cache.fetch(['project_dashboard', project], expires_in: EXPIRES_IN) do
```

Comment:
worth invalidating the cache with a webhook https://docs.rollbar.com/docs/webhooks ?

Reply:
nah ... 1 minute cache is fine